index.html

```diff
@@ -20,160 +20,166 @@
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+
+      <button>Donate Now</button>
+    </header>
+
+    <content class="content">
+      <main class="main">
+        <main class="alert">
+          You've helped us give <span class="text-highlight">72</span> bikes to
+          refugees and asylum seekers so far. <strong>Thank you!</strong>
+        </main>
+
+        <figcaption class="hero">
+          <h1>Bikes for Refugees</h1>
+          <p>
+            Providing donated bikes and accessories to refugees and asylum
+            seekers in Scotland.
+          </p>
+          <button>Donate a bike today</button>
+          <button class="btn">Volunteer</button>
+        </figcaption>
+
+        <article>
+          <h2 class="heading-underline">Learn more</h2>
+
+          <article class="learn-more-articles">
+            <section class="block">
+              <h3 class="article__title">Why do refugees need bikes?</h3>
+              <p class="article__summary">
+                Many refugees are placed in housing in areas where there are few
+                jobs. Bikes provide low-cost transportation so that they can get
+                to work, manage child care and do their shopping.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
+              </p>
+            </section>
+
+            <section class="block">
+              <h3 class="article__title">How can I help?</h3>
+              <p class="article__summary">
+                We need lots of bikes and bike accessories! If you have an old
+                bike you don't use, bring it to one of our dropoff events. Or
+                come attend one of our fundraisers.
+              </p>
+              <p class="article__read-more">
+                <a href="#">
+                  <span class="arrow-right"></span>
+                  Learn more
+                </a>
+              </p>
+            </section>
+          </article>
+        </article>
+      </main>
+
+      <article class="sidebar">
+        <h2 class="heading-underline">Upcoming events</h2>
 
-        <a href="#">
+        <aside class="article">
           <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
+            class="article__thumbnail"
+            src="images/spring-fundraisers_thumbnail.jpg"
+            alt=""
```

Comment:
There is no alt tag here!
Also there are some comments in the css folder about this image styling. Pls check the css file!